src/main/java/mat/server/fhirvalidation/FhirValidationReportServlet.java

```diff
@@ -0,0 +1,49 @@
+package mat.server.fhirvalidation;
+
+import freemarker.template.TemplateException;
+import mat.server.service.FhirValidationReportService;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.http.MediaType;
+import org.springframework.web.context.support.WebApplicationContextUtils;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class FhirValidationReportServlet extends HttpServlet {
```

Comment:
Since MAT is a Spring WebMVC application we might prefer to go with a "@Controller" instead of a plain servlet. It should be a small change

Reply:
Right now I don't see WebMvc enabled. Do you want me to add configuration for this and modify this accordingly?

Reply:
Right. I was under impression SpingMVC is fully configured. We will have to re-visit it later..